101-bundle-json.md

```diff
@@ -333,6 +334,62 @@ Some validation terms have aliased keywords for backward compatibility with ARM
 
 > The term _parameters_ indicates the present specification of what can be provided to a bundle. The term _values_ is frequently used to indicate the user-supplied values which are tested against the parameter definitions.
 
+### Format of Parameter Specification
+
+The structure of a parameters section looks like this:
+
+```json
+"parameters": {
+    "<parameter-name>" : {
+        "type" : "<type-of-parameter-value>",
+        "schema": "<reference-to-schema-document>",
+        "required": true|false
+        "defaultValue": "<default-value-of-parameter>",
+        "allowedValues": [ "<array-of-allowed-values>" ],
+        "minValue": <minimum-value-for-int>,
+        "maxValue": <maximum-value-for-int>,
+        "minimum": <minimum-length-for-string-or-array>,
+        "maximum": <maximum-length-for-string-or-array-parameters>,
+        "metadata": {
+            "description": "<description-of-the parameter>"
+        },
+        "destination": {
+            "env": "<name-of-env-var>",
+            "path": "<fully-qualified-path>"
+        },
+        "apply-to": ["action1", "action2"]
+    }
+}
+```
+
+See [The Bundle Runtime](103-bundle-runtime.md) for details of how parameters are injected into the invocation image.
+
+### Specifying Schema References
```

Comment:
Thoughts on adding schema support to credentials, too? Azure credential files, for example, are schematized JSON. (Definitely the kind of thing we could defer to a follow-up PR)